acceptance/localcluster/docker.go

```diff
@@ -0,0 +1,295 @@
+// Copyright 2015 The Cockroach Authors.
```

Comment:
There are various `dockerutil` packages out there. They provide a mixed-bag of functionality that seems tailored to their parent project, just like ours is.